docs/products/kafka/kafka-mirrormaker/concepts/mirrormaker2-tuning.rst

```diff
@@ -8,22 +8,32 @@ MirrorMaker 2 common parameters
     * ``kafka_mirrormaker.emit_checkpoints_interval_seconds``
     * ``kafka_mirrormaker.sync_group_offsets_interval_seconds`` 
     * Replication flow:
-        * Sync interval in seconds
+        * ``Sync interval in seconds``
 
-3. Add `*[\-\.]` internal to the topic black list:
-    * connect.* __.*, crowdcontrol.public.events, .*[\-\.]internal
-    * The following parameters can be tuned depending on use case:
+3. Add internal topics to the topic black list:
```

Comment:
Is "blacklist" the term used in the UI? If not, could we change this to "denylist" (or another rewording, as appropriate) please? For reference, see the [style guide entry for blacklist](https://developers.google.com/style/word-list#blacklist).

Reply:
@staceysalamon-aiven 
I actually thought about this as I was updating, unfortunately it is in the UI, should we update it for now but make a change request to our console UI?
![image](https://github.com/aiven/devportal/assets/23372934/0123f690-420e-46c0-94c4-d5201b06fe37)